storybookjs/storybook
# fix: Vue example is missing a vue-template-compiler dependency

Issue:

## What I did
fix: Vue example is missing a vue-template-compiler dependency

## How to test
cd examples/vue-kitchen-sink
npm i
npm run storybook


If your answer is yes to any of these, please make sure to include it in your PR.
